### PR TITLE
docs api register

### PR DESCRIPTION
**Pull request by ghost**
## Description
docs api register
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-thaitran** [comment]
LGTM

File: docs/api-references/user.md [hunk 1]
```diff
@@ -112,4 +112,63 @@ Logout User
...
+        "data": {
+            "id": 1,
+            "name": "abcxyz",
+            "email": "abcxyz@gmail.com",
+            "role": 1,
+            "user_info": {
+                "id": 1,
```
Comment (at-thaitran) [inline docs/api-references/user.md]: thiếu created_at, updated_at, deleted_at

File: docs/api-references/user.md [hunk 2]
```diff
@@ -112,4 +112,63 @@ Logout User
...
+                "user_id": 1,
+                "address": "12 DN",
+                "phone": "0913",
+                "avatar": "default-user-avatar.png",
```
Comment (at-thaitran) [inline docs/api-references/user.md]: thiếu created_at, updated_at, deleted_at

File: docs/api-references/user.md [hunk 3]
```diff
@@ -112,4 +112,63 @@ Logout User
...
+        "name": [
+            "The name field is required."
+        ],
+        "email": [
+            "The email has already been taken."
+        ],
+    },
+    "code": 422,
+    "request": {
+        "name": "abcxyz",
```
Comment (at-thaitran) [inline docs/api-references/user.md]: ở trên `required` required mà sao `$request->name` lại có

File: docs/api-references/user.md [hunk 4]
```diff
@@ -112,4 +112,63 @@ Logout User
...
+        "data": {
+            "id": 1,
+            "name": "abcxyz",
+            "email": "abcxyz@gmail.com",
+            "role": 1,
```
Comment (at-thaitran) [inline docs/api-references/user.md]: đâu cần `role`

File: docs/api-references/user.md [hunk 5]
```diff
@@ -112,4 +112,59 @@ Logout User
...
+            "user_info": {
+                "id": 1,
+                "user_id": 1,
+                "address": "12 DN",
+                "phone": "0913",
+                "avatar": "default-user-avatar.png",
+                "avatar_url": "http://192.168.33.10/images/users/default-user-avatar.png"
+            }
+        }
+    },
```
Comment (at-vietpham) [inline docs/api-references/user.md]: Sai cấu trúc JSON. dùng https://jsonformatter.curiousconcept.com/ để check

File: docs/api-references/user.md [hunk 6]
```diff
@@ -112,4 +112,59 @@ Logout User
...
+        ],
+    },
+    "code": 422,
+    "request": {
+        "name": "abcxyz",
+        "email": "abcxyz@gmail.com",
+        "role": 1,
+        "password": "123456789",
+        "address": "12 Da Nang",
+        "phone": "0913",
```
Comment (at-vietpham) [inline docs/api-references/user.md]: Sai cấu trúc JSON. dùng https://jsonformatter.curiousconcept.com/ để check

File: docs/api-references/user.md [hunk 7]
```diff
@@ -112,4 +112,59 @@ Logout User
...
+```
+/api/register
+```
+Register user
+
+#### Request Headers
+| Key | Value |
+|---|---|
+|Accept|application/json
+
```
Comment (at-vietpham) [inline docs/api-references/user.md]: Bổ dung table mô tả Parameters

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**